### ISSUE TITLE
I found an issue probably many people have same situation like me 

### ISSUE THREAD

**Issue by tori-hotaru** [kohya-ss/sd-scripts#161]
Device: win10 / 3060ti
In my training step, i use the notepad++ to change the parameters, however, when i used shell to activate, it gave me this error.(picture 1)
![Q5P10SX7W5TW}G4HD8}W (V](https://user-images.githubusercontent.com/58972377/217159145-602dd0fb-23c4-4d9d-8590-df9fe12ff79c.png)
I quickly found that there is something wrong in my shell profile. Althought the parameter "--learning-rate" has number in it , i dont know why the programme ignored it. So I changed the programme like this
![E(0OA`QZV(6XS@$HZ5 `DC](https://user-images.githubusercontent.com/58972377/217159291-6d0856a6-e34d-4ad5-957b-3d6c8154663b.png)
Fortunately, it worked, but then it reported me another error like this.
![)@J1F(1MWWKV B N$I{`MKQ](https://user-images.githubusercontent.com/58972377/217159988-2fe6cc46-2cb8-4914-817d-9d48d0712cc6.png)
I dont know if it is because the pc language differences or other reason.So is there any better way to solve those problems? I was quite confused.
By the way thanks for your programme, it looks very cool !

**Comment by rockerBOO** [comment]
toss a quote around the learning rate `--learning-rate="1e-4"` and see if that helps
